teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<p>Use</p><p><code>DATA / "asparin.png"</code></p><p>instead of </p><p><code>'./data/aspirin.png'</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>done</p>